ramandecompy/dataprep.py

```diff
@@ -9,10 +9,8 @@
 
```

Comment:
good description, would be good to point to juypter notebook example here

Reply:
No the spectrafit one got merged. I am putting the doc stings on now as well as troubleshooting a bug that will make it so that experiment filenames that have extra "." in them such as 7.5s will still work with the add_experiment function

Reply:
Also sorry this comment is in the wrong spot